### ISSUE TITLE
Maybe requires xsel on ubuntu groovy Wayland session

### ISSUE THREAD

**Issue by cbs228** [rolandwalker/simpleclip#18]
On Ubuntu 20.10, I find it necessary to

```bash
sudo apt install xsel
```

in order to get paste support to work. Without `xsel`, only copy works.

Can anyone else replicate this?

**Comment by rolandwalker** [comment]
Hi!  This happens in the GUI?

**Comment by cbs228** [comment]
Yes, this happens when launching emacs via the "Emacs (gui)" option or `emacs-gtk`.

**Comment by fuzzbomb** [comment]
Replicated. The package assumes the `xsel` program is present, however it isn't installed by default in all Linux/X11 distros.

For example: using the Arch Linux installer, and requesting a KDE Plasma desktop, installs X11 and Wayland, but doesn't install `xsel`.

The README could do with improvement; the "no external dependencies" is a bit misleading here.
